apps/trading/public/reward-cards.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "title": "Reward A",
+    "description": "AAA This is a description that returns onto a maximum number of lines to be defined. It may also be truncated at 3 or 4 lines.\n\n* **1,666.58 NEB**\n* Distribution strategy: Pro rata \n* Reward pool amount and asset\n* Another bullet point\n* Another bullet point",
```

Comment:
Having to write markdown in to a json string kind of defeats the purpose of using markdown as the editing experience is poor.

Its probably fine for now but I think it would probably be better to just have some .mdx files in the repo to import